test/index.test.js

```diff
@@ -1,9 +1,323 @@
 'use strict';
 
+const Hapi = require('hapi');
 const assert = require('chai').assert;
+const mockery = require('mockery');
+const sinon = require('sinon');
 
-describe('index test', () => {
-    it('fails', () => {
-        assert.isTrue(false);
+require('sinon-as-promised');
+
+sinon.assert.expose(assert, { prefix: '' });
+
+/**
+ * helper to generate a nodemailer mock
+ * @method getUserMock
+ * @return {Function}         Stubbed function
+ */
+function getNodemailerMock() {
+    const sendMailMock = {
+        sendMail: sinon.stub().yieldsAsync()
+    };
+
+    return { createTransport: sinon.stub().returns(sendMailMock) };
+}
+
+describe('index', () => {
+    let serverMock;
+    let configMock;
+    let notifier;
+    let buildDataMock;
+    let nodemailerMock;
+
+    describe('notifier listens to server emits', () => {
+        before(() => {
```

Comment:
You are doing almost the exact same setup and teardown in every describe. Perhaps you can put these to a higher level describe.